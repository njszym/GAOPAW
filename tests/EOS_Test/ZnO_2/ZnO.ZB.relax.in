&control
  calculation='relax'
  prefix='ZnO.ZB.relax'
  pseudo_dir='./'
/
&system
  ibrav=0
  nat=2
  ntyp=2
  ecutwfc=40
  ecutrho=200
  smearing='fermi-dirac'
  degauss=0.002
  occupations='smearing'
/
&electrons
  conv_thr=1.d-8
/
&ions
  ion_dynamics='bfgs'
/
&cell
  cell_dynamics='bfgs'
/
K_POINTS (automatic)
  8 8 8 0 0 0
ATOMIC_SPECIES
  Zn 0.0 Zn.GGA-PBE-paw.UPF
  O 0.0 O.GGA-PBE-paw.UPF
ATOMIC_POSITIONS crystal 
Zn      -0.000000000  -0.000000000   0.000000000
O        0.250000000   0.250000000   0.250000000
CELL_PARAMETERS bohr
-4.303161890882344 0.0 4.303161890882344 
-0.0 4.303161890882344 4.303161890882344 
-4.303161890882344 4.303161890882344 0.0 
