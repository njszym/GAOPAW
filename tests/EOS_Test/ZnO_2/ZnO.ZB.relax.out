===== VARIABLES =====
EXE_PATH = /scr/szymansk/2019-3/qe-bin/bin/pw.x
tpp = 4
Parallel keywords = -nimage 1
MPIRUN_EXE = orterun
PATH = /scr/szymansk/2019-3/mmshare-v4.7/lib/Linux-x86_64/openmpi/bin:/scr/szymansk/2019-3/internal/bin:/scr/szymansk/2019-3/mmshare-v4.7/bin/Linux-x86_64:/home/szymansk/bin:/scr/szymansk/dakota/bin:/scr/szymansk/atompaw-4.1.0.5/src/:/usr/lib64/qt-3.3/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:/sbin
LD_LIBRARY_PATH = /scr/szymansk/2019-3/mmshare-v4.7/lib/Linux-x86_64/openmpi/lib:/scr/szymansk/2019-3/internal/lib/ssl:/scr/szymansk/2019-3/internal/lib:/scr/szymansk/2019-3/mmshare-v4.7/lib/Linux-x86_64:/scr/szymansk/2019-3/internal/lib/python3.6/site-packages:/scr/szymansk/dakota/bin:/software/lib/Linux-x86_64/ifort_2017.1.132/mkl/lib/intel64
===== END VARIABLES =====
Invoking: exe_cmd () 
{ 
    "$MPIRUN_EXE" -np $tpp "$EXE_PATH" -in $input_file "${P_KEYWORDS[@]}"
}

     Program PWSCF v.6.3MaX starts on 13Jun2019 at 12:18:45 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     4 processors

     MPI processes distributed on     1 nodes
     R & G space division:  proc/nbgrp/npool/nimage =       4
     Reading input from ZnO.ZB.relax.in
Warning: card &CELL ignored
Warning: card   CELL_DYNAMICS='BFGS' ignored
Warning: card / ignored

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min         127     105     31                 1907     1394     241
     Max         128     106     32                 1909     1395     242
     Sum         511     421    127                 7631     5577     965
 


     bravais-lattice index     =            0
     lattice parameter (alat)  =       6.0856  a.u.
     unit-cell volume          =     159.3650 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            2
     number of electrons       =        26.00
     number of Kohn-Sham states=           17
     kinetic-energy cutoff     =      40.0000  Ry
     charge density cutoff     =     200.0000  Ry
     convergence threshold     =      1.0E-08
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      = SLA  PW   PBE  PBE ( 1  4  3  4 0 0)
     nstep                     =           50


     celldm(1)=   6.085590  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (  -0.707107   0.000000   0.707107 )  
               a(2) = (   0.000000   0.707107   0.707107 )  
               a(3) = (  -0.707107   0.707107   0.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = ( -0.707107 -0.707107  0.707107 )  
               b(2) = (  0.707107  0.707107  0.707107 )  
               b(3) = ( -0.707107  0.707107 -0.707107 )  


     PseudoPot. # 1 for Zn read from file:
     ./Zn.GGA-PBE-paw.UPF
     MD5 check sum: df62231357ef9e81f77b2b3087fa5675
     Pseudo is Ultrasoft, Zval = 20.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  943 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2
     Q(r) pseudized with  8 coefficients,  rinner =    1.100   1.100   1.100
                                                       1.100   1.100

     PseudoPot. # 2 for O  read from file:
     ./O.GGA-PBE-paw.UPF
     MD5 check sum: 734c27235a0248c51dbae37a1fbe46ec
     Pseudo is Ultrasoft + core correction, Zval =  6.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  737 points,  5 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
     Q(r) pseudized with  8 coefficients,  rinner =    0.900   0.900   0.900
                                                       0.900   0.900

     atomic species   valence    mass     pseudopotential
        Zn            20.00    65.39000     Zn( 1.00)
        O              6.00    15.99940     O ( 1.00)

     24 Sym. Ops. (no inversion) found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Zn  tau(   1) = (   0.0000000   0.0000000   0.0000000  )
         2           O   tau(   2) = (  -0.3535534   0.3535534   0.3535534  )

     number of k points=    29  Fermi-Dirac smearing, width (Ry)=  0.0020
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.0039062
        k(    2) = (  -0.0883883   0.0883883  -0.0883883), wk =   0.0312500
        k(    3) = (  -0.1767767   0.1767767  -0.1767767), wk =   0.0312500
        k(    4) = (  -0.2651650   0.2651650  -0.2651650), wk =   0.0312500
        k(    5) = (   0.3535534  -0.3535534   0.3535534), wk =   0.0156250
        k(    6) = (   0.0000000   0.1767767   0.0000000), wk =   0.0234375
        k(    7) = (  -0.0883883   0.2651650  -0.0883883), wk =   0.0937500
        k(    8) = (  -0.1767767   0.3535534  -0.1767767), wk =   0.0937500
        k(    9) = (   0.4419417  -0.2651650   0.4419417), wk =   0.0937500
        k(   10) = (   0.3535534  -0.1767767   0.3535534), wk =   0.0937500
        k(   11) = (   0.2651650  -0.0883883   0.2651650), wk =   0.0937500
        k(   12) = (   0.1767767   0.0000000   0.1767767), wk =   0.0468750
        k(   13) = (   0.0000000   0.3535534   0.0000000), wk =   0.0234375
        k(   14) = (  -0.0883883   0.4419417  -0.0883883), wk =   0.0937500
        k(   15) = (   0.5303301  -0.1767767   0.5303301), wk =   0.0937500
        k(   16) = (   0.4419417  -0.0883883   0.4419417), wk =   0.0937500
        k(   17) = (   0.3535534   0.0000000   0.3535534), wk =   0.0468750
        k(   18) = (   0.0000000   0.5303301   0.0000000), wk =   0.0234375
        k(   19) = (   0.6187184  -0.0883883   0.6187184), wk =   0.0937500
        k(   20) = (   0.5303301   0.0000000   0.5303301), wk =   0.0468750
        k(   21) = (   0.0000000  -0.7071068   0.0000000), wk =   0.0117188
        k(   22) = (  -0.1767767   0.3535534   0.0000000), wk =   0.0937500
        k(   23) = (   0.4419417  -0.2651650   0.6187184), wk =   0.1875000
        k(   24) = (   0.3535534  -0.1767767   0.5303301), wk =   0.0937500
        k(   25) = (   0.5303301  -0.1767767   0.7071068), wk =   0.0937500
        k(   26) = (   0.4419417  -0.0883883   0.6187184), wk =   0.1875000
        k(   27) = (   0.3535534   0.0000000   0.5303301), wk =   0.0937500
        k(   28) = (  -0.1767767  -0.7071068   0.0000000), wk =   0.0468750
        k(   29) = (  -0.3535534  -0.7071068   0.0000000), wk =   0.0234375

     Dense  grid:     7631 G-vectors     FFT dimensions: (  27,  27,  27)

     Smooth grid:     5577 G-vectors     FFT dimensions: (  25,  25,  25)

     Estimated max dynamical RAM per process >      10.36 MB

     Estimated total dynamical RAM >      41.43 MB

     Check: negative/imaginary core charge=   -0.000003    0.000000

     Initial potential from superposition of free atoms

     starting charge   25.99986, renormalised to   26.00000
     Starting wfcs are   17 randomized atomic wfcs

     total cpu time spent up to now is        0.2 secs

     Self-consistent Calculation

     iteration #  1     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  3.0

     total cpu time spent up to now is        0.5 secs

     total energy              =    -493.69476727 Ry
     Harris-Foulkes estimate   =    -494.02920439 Ry
     estimated scf accuracy    <       0.43691151 Ry

     iteration #  2     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.68E-03,  avg # of iterations =  2.1

     total cpu time spent up to now is        0.7 secs

     total energy              =    -493.76670658 Ry
     Harris-Foulkes estimate   =    -494.10379526 Ry
     estimated scf accuracy    <       0.75934157 Ry

     iteration #  3     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.68E-03,  avg # of iterations =  2.0

     total cpu time spent up to now is        1.0 secs

     total energy              =    -493.91100364 Ry
     Harris-Foulkes estimate   =    -493.91376285 Ry
     estimated scf accuracy    <       0.00537880 Ry

     iteration #  4     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.07E-05,  avg # of iterations =  2.0

     total cpu time spent up to now is        1.2 secs

     total energy              =    -493.91230521 Ry
     Harris-Foulkes estimate   =    -493.91298665 Ry
     estimated scf accuracy    <       0.00170713 Ry

     iteration #  5     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  6.57E-06,  avg # of iterations =  2.0

     total cpu time spent up to now is        1.5 secs

     total energy              =    -493.91259430 Ry
     Harris-Foulkes estimate   =    -493.91261016 Ry
     estimated scf accuracy    <       0.00003679 Ry

     iteration #  6     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.41E-07,  avg # of iterations =  1.5

     total cpu time spent up to now is        1.7 secs

     total energy              =    -493.91259869 Ry
     Harris-Foulkes estimate   =    -493.91259832 Ry
     estimated scf accuracy    <       0.00000043 Ry

     iteration #  7     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.64E-09,  avg # of iterations =  2.0

     total cpu time spent up to now is        1.9 secs

     total energy              =    -493.91259878 Ry
     Harris-Foulkes estimate   =    -493.91259879 Ry
     estimated scf accuracy    <       0.00000003 Ry

     iteration #  8     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.30E-10,  avg # of iterations =  1.1

     total cpu time spent up to now is        2.1 secs

     End of self-consistent calculation

          k = 0.0000 0.0000 0.0000 (   701 PWs)   bands (ev):

  -114.5183 -68.7140 -68.7140 -68.7140  -9.1559   3.1431   3.1431   3.1431
     4.4676   4.4676   8.9442   8.9442   8.9442   9.6587  22.4438  22.4438
    22.4438

          k =-0.0884 0.0884-0.0884 (   679 PWs)   bands (ev):

  -114.5183 -68.7144 -68.7141 -68.7141  -9.0569   3.0637   3.1907   3.1907
     4.4605   4.4605   7.9450   8.8291   8.8291  10.7582  21.2228  22.5688
    22.5688

          k =-0.1768 0.1768-0.1768 (   670 PWs)   bands (ev):

  -114.5182 -68.7155 -68.7143 -68.7143  -8.8096   2.9605   3.2956   3.2956
     4.4665   4.4665   6.6432   8.5474   8.5474  12.2790  19.2324  22.6004
    22.6004

          k =-0.2652 0.2652-0.2652 (   676 PWs)   bands (ev):

  -114.5181 -68.7165 -68.7145 -68.7145  -8.5494   3.0521   3.3783   3.3783
     4.5177   4.5177   5.4209   8.2539   8.2539  13.7253  17.6448  22.3115
    22.3115

          k = 0.3536-0.3536 0.3536 (   700 PWs)   bands (ev):

  -114.5181 -68.7169 -68.7145 -68.7145  -8.4376   3.2292   3.4038   3.4038
     4.5564   4.5564   4.8059   8.1265   8.1265  14.4767  16.9044  22.1395
    22.1395

          k = 0.0000 0.1768 0.0000 (   673 PWs)   bands (ev):

  -114.5183 -68.7144 -68.7141 -68.7141  -9.0243   3.0033   3.2279   3.2279
     4.4217   4.4898   7.9733   8.6464   8.6464  11.0397  21.2803  22.4244
    22.4244

          k =-0.0884 0.2652-0.0884 (   674 PWs)   bands (ev):

  -114.5182 -68.7152 -68.7144 -68.7143  -8.8272   2.8990   3.3244   3.3388
     4.3953   4.5111   7.0499   8.3132   8.3958  12.2372  19.8471  22.0430
    22.4766

          k =-0.1768 0.3536-0.1768 (   676 PWs)   bands (ev):

  -114.5181 -68.7162 -68.7146 -68.7145  -8.5711   2.9639   3.4100   3.4410
     4.4478   4.5475   5.8412   8.0141   8.0435  13.7062  18.2122  21.3526
    22.4061

          k = 0.4419-0.2652 0.4419 (   679 PWs)   bands (ev):

  -114.5181 -68.7167 -68.7148 -68.7146  -8.4085   3.2213   3.4670   3.4770
     4.5590   4.5761   4.8857   7.7122   7.9328  14.9548  16.9729  21.0281
    22.3067

          k = 0.3536-0.1768 0.3536 (   676 PWs)   bands (ev):

  -114.5181 -68.7166 -68.7148 -68.7145  -8.4452   3.2329   3.3781   3.4506
     4.4932   4.5816   5.1454   7.6004   8.1310  14.6820  17.2921  20.8601
    22.4715

          k = 0.2652-0.0884 0.2652 (   674 PWs)   bands (ev):

  -114.5182 -68.7159 -68.7146 -68.7143  -8.6567   2.9877   3.3509   3.3656
     4.4236   4.5372   6.2277   7.8078   8.4755  13.2752  18.8078  20.9164
    22.9091

          k = 0.1768 0.0000 0.1768 (   670 PWs)   bands (ev):

  -114.5182 -68.7150 -68.7143 -68.7141  -8.9086   2.9766   3.2527   3.2746
     4.4230   4.4900   7.2926   8.3148   8.7598  11.7829  20.3642  21.6174
    22.9770

          k = 0.0000 0.3536 0.0000 (   681 PWs)   bands (ev):

  -114.5182 -68.7155 -68.7146 -68.7146  -8.6991   2.7573   3.4764   3.4764
     4.3334   4.5435   7.0343   7.9009   7.9009  12.8068  19.4274  22.3160
    22.6225

          k =-0.0884 0.4419-0.0884 (   681 PWs)   bands (ev):

  -114.5181 -68.7161 -68.7149 -68.7148  -8.4952   2.7651   3.6145   3.6207
     4.3734   4.5731   6.2638   7.4641   7.5373  13.8459  18.5646  20.6861
    22.7432

          k = 0.5303-0.1768 0.5303 (   684 PWs)   bands (ev):

  -114.5181 -68.7165 -68.7152 -68.7148  -8.3350   2.9936   3.6760   3.7669
     4.5699   4.5839   5.2034   6.9692   7.4502  15.1907  17.5648  19.9852
    22.7138

          k = 0.4419-0.0884 0.4419 (   687 PWs)   bands (ev):

  -114.5181 -68.7164 -68.7155 -68.7147  -8.3175   3.2113   3.6273   3.8146
     4.4940   4.5126   5.0378   6.6371   7.7090  16.2291  16.7997  19.5062
    22.7163

          k = 0.3536 0.0000 0.3536 (   676 PWs)   bands (ev):

  -114.5181 -68.7161 -68.7153 -68.7145  -8.4529   3.2493   3.3428   3.5035
     4.4240   4.5848   5.7485   6.8457   8.1354  14.8135  18.3376  19.2394
    22.8795

          k = 0.0000 0.5303 0.0000 (   684 PWs)   bands (ev):

  -114.5181 -68.7164 -68.7150 -68.7150  -8.3640   2.6533   3.8284   3.8284
     4.3508   4.5974   6.1880   7.0763   7.0763  13.9758  18.2012  20.7165
    23.0645

          k = 0.6187-0.0884 0.6187 (   682 PWs)   bands (ev):

  -114.5181 -68.7166 -68.7153 -68.7151  -8.2565   2.7557   3.9178   3.9717
     4.5282   4.5960   5.6191   6.6011   6.9246  14.6007  17.9533  20.1986
    23.1323

          k = 0.5303 0.0000 0.5303 (   682 PWs)   bands (ev):

  -114.5180 -68.7164 -68.7157 -68.7150  -8.2384   2.9932   3.8254   4.1840
     4.5380   4.7261   4.8864   6.2477   7.2067  15.6498  17.5604  19.5808
    22.9639

          k = 0.0000-0.7071 0.0000 (   692 PWs)   bands (ev):

  -114.5181 -68.7168 -68.7152 -68.7152  -8.2225   2.6578   4.0339   4.0339
     4.4769   4.6198   5.7004   6.6817   6.6817  14.2246  17.9382  20.3480
    23.3108

          k =-0.1768 0.3536 0.0000 (   676 PWs)   bands (ev):

  -114.5182 -68.7158 -68.7148 -68.7145  -8.6260   2.8800   3.4160   3.4752
     4.3849   4.5517   6.4790   7.5416   8.1986  13.4043  19.1758  20.4428
    22.8404

          k = 0.4419-0.2652 0.6187 (   681 PWs)   bands (ev):

  -114.5181 -68.7163 -68.7151 -68.7147  -8.4182   3.0882   3.4409   3.6434
     4.4583   4.5873   5.5365   7.0672   7.8471  14.8726  17.9102  19.8791
    22.5159

          k = 0.3536-0.1768 0.5303 (   682 PWs)   bands (ev):

  -114.5181 -68.7165 -68.7152 -68.7148  -8.3301   3.2592   3.4721   3.8148
     4.4700   4.6884   4.8287   6.9889   7.6371  16.0554  16.7232  19.9527
    22.2913

          k = 0.5303-0.1768 0.7071 (   680 PWs)   bands (ev):

  -114.5181 -68.7164 -68.7152 -68.7149  -8.3398   2.8513   3.6765   3.8613
     4.4600   4.5919   5.8058   6.7471   7.2618  14.7355  18.1518  19.9830
    22.3529

          k = 0.4419-0.0884 0.6187 (   687 PWs)   bands (ev):

  -114.5181 -68.7163 -68.7156 -68.7150  -8.2527   3.0748   3.7169   4.0656
     4.4642   4.7727   5.0816   6.3366   7.1010  15.8633  17.6522  19.6136
    21.7490

          k = 0.3536 0.0000 0.5303 (   684 PWs)   bands (ev):

  -114.5181 -68.7162 -68.7157 -68.7149  -8.2859   3.2110   3.5614   4.0176
     4.4127   4.6055   5.2482   6.3071   7.4089  16.1783  17.4445  19.2008
    22.0576

          k =-0.1768-0.7071 0.0000 (   682 PWs)   bands (ev):

  -114.5180 -68.7166 -68.7154 -68.7152  -8.2207   2.8568   3.9767   3.9873
     4.5606   4.6855   5.4306   6.3256   6.7677  15.0644  17.9424  20.1814
    21.8534

          k =-0.3536-0.7071 0.0000 (   684 PWs)   bands (ev):

  -114.5180 -68.7162 -68.7158 -68.7152  -8.2189   3.1498   3.8594   3.9383
     4.4629   4.9644   5.2185   5.9013   6.8243  16.2073  17.9482  20.0632
    20.2598

     the Fermi energy is     9.3175 ev

!    total energy              =    -493.91259878 Ry
     Harris-Foulkes estimate   =    -493.91259878 Ry
     estimated scf accuracy    <          4.0E-10 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =    -401.34054917 Ry
     hartree contribution      =     211.69128596 Ry
     xc contribution           =     -49.62912090 Ry
     ewald contribution        =    -254.63421467 Ry
     smearing contrib. (-TS)   =      -0.00000000 Ry

     convergence has been achieved in   8 iterations

     Forces acting on atoms (cartesian axes, Ry/au):

     atom    1 type  1   force =     0.00000000    0.00000000   -0.00000000
     atom    2 type  2   force =    -0.00000000   -0.00000000    0.00000000

     Total force =     0.000000     Total SCF correction =     0.000000

     BFGS Geometry Optimization

     bfgs converged in   1 scf cycles and   0 bfgs steps
     (criteria: energy <  1.0E-04 Ry, force <  1.0E-03 Ry/Bohr)

     End of BFGS Geometry Optimization

     Final energy   =    -493.9125987807 Ry
Begin final coordinates

ATOMIC_POSITIONS (crystal)
Zn       0.000000000   0.000000000   0.000000000
O        0.250000000   0.250000000   0.250000000
End final coordinates



     Writing output data file ZnO.ZB.relax.save/
 
     init_run     :      0.13s CPU      0.16s WALL (       1 calls)
     electrons    :      1.77s CPU      1.89s WALL (       1 calls)
     forces       :      0.03s CPU      0.03s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.06s CPU      0.09s WALL (       1 calls)
     potinit      :      0.00s CPU      0.00s WALL (       1 calls)
     hinit0       :      0.06s CPU      0.06s WALL (       1 calls)

     Called by electrons:
     c_bands      :      1.40s CPU      1.50s WALL (       8 calls)
     sum_band     :      0.30s CPU      0.31s WALL (       8 calls)
     v_of_rho     :      0.02s CPU      0.02s WALL (       9 calls)
     newd         :      0.05s CPU      0.05s WALL (       9 calls)
     mix_rho      :      0.00s CPU      0.00s WALL (       8 calls)

     Called by c_bands:
     init_us_2    :      0.03s CPU      0.03s WALL (     522 calls)
     cegterg      :      1.29s CPU      1.39s WALL (     232 calls)

     Called by sum_band:
     sum_band:bec :      0.00s CPU      0.00s WALL (     232 calls)
     addusdens    :      0.04s CPU      0.04s WALL (       8 calls)

     Called by *egterg:
     h_psi        :      0.97s CPU      1.05s WALL (     716 calls)
     s_psi        :      0.02s CPU      0.02s WALL (     716 calls)
     g_psi        :      0.00s CPU      0.00s WALL (     455 calls)
     cdiaghg      :      0.23s CPU      0.27s WALL (     687 calls)

     Called by h_psi:
     h_psi:pot    :      0.97s CPU      1.05s WALL (     716 calls)
     h_psi:calbec :      0.03s CPU      0.04s WALL (     716 calls)
     vloc_psi     :      0.91s CPU      0.98s WALL (     716 calls)
     add_vuspsi   :      0.02s CPU      0.02s WALL (     716 calls)

     General routines
     calbec       :      0.05s CPU      0.06s WALL (    1064 calls)
     fft          :      0.02s CPU      0.02s WALL (     127 calls)
     ffts         :      0.00s CPU      0.00s WALL (      17 calls)
     fftw         :      1.00s CPU      1.08s WALL (   23320 calls)
     interpolate  :      0.00s CPU      0.00s WALL (       9 calls)
 
     Parallel routines
     fft_scatt_xy :      0.13s CPU      0.14s WALL (   23464 calls)
     fft_scatt_yz :      0.40s CPU      0.43s WALL (   23464 calls)
 
     PWSCF        :     1.99s CPU         2.19s WALL

 
   This run was terminated on:  12:18:47  13Jun2019            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
